pulpcore/app/util.py

```diff
@@ -571,19 +536,58 @@ def __getattr__(self, *args, **kwargs):
             return self
 
     @classmethod
-    def build(cls, domain):
+    def build(cls, *args, **kwargs):
         otel_enabled = os.getenv("PULP_OTEL_ENABLED")
```

Comment:
What if you change here to:
```suggestion
        otel_enabled = os.getenv("PULP_OTEL_ENABLED", "").lower() == "true"
```

It will set `otel_enabled` as a boolean. Then on 541 you can just check for `otel_enabled`.

Reply:
Done! 🎉